### PR TITLE
New admin notice UI based on user list example

### PR DESCRIPTION
**Pull request by sapk**
capture d ecran_2015-09-25_18-28-06

### CODE REVIEW

**Comment by Unknwon** [comment]
Thanks and one little thing: total number can be showed after `system notice` title, like in users page.

**Comment by sapk** [comment]
Ok i will correct it

File: routers/admin/notice.go [hunk 1]
```diff
@@ -6,11 +6,13 @@ package admin
 
 import (
 	"github.com/Unknwon/com"
+        "github.com/Unknwon/paginater"
```
Comment (TheHowl) [inline routers/admin/notice.go]: Indentation seems a bit fucked up here. Run `go fmt`.

**Comment by Unknwon** [comment]
It is interesting that I keep getting build fail notices where here is passed: https://travis-ci.org/gogits/gogs/builds/82186221 :joy:

**Comment by Unknwon** [comment]
Looks good, merging!

**Comment by sapk** [comment]
Maybe you receive fail from the travis running on my fork (since it show in gitter) : https://travis-ci.org/sapk/gogs/jobs/82186216 since it pull (go get) from my master that is outdated. Maybe ?

**Comment by Unknwon** [comment]
@sapk possible.

**Pull request merged**